frontends/mit-open/src/page-components/ChannelDetails/ChannelDetails.tsx

```diff
@@ -0,0 +1,187 @@
+          label: "Free",
+        },
+      ],
+      name: "free",
+    },
+    {
+      name: "topic",
+      title: "Topic",
+    },
+    {
+      name: "formats",
+      title: "Formats",
+    },
+    {
+      name: "fee",
+      title: "Fee",
+    },
+    {
+      name: "department",
+      title: "Department",
+    },
+    {
+      name: "offerings",
+      title: "Offerings",
+    },
+    {
+      name: "level",
+      title: "Level",
+    },
+    {
+      name: "content_types",
+      title: "Type of Content",
+    },
+    {
+      name: "audience",
+      title: "Audience",
+    },
+    {
+      name: "more_information",
+      title: "More Information",
+      labelFunction: (key: string, channelTitle: string) => (
+        <a href={key}>
+          {channelTitle} website <OpenInNewIcon fontSize="inherit" />
+        </a>
+      ),
+    },
+    {
+      name: "platform",
+      title: "Platform",
+    },
+    {
+      name: "offered_by",
+      title: "Offered By",
+    },
+    {
+      name: "certifications",
+      title: "Certificate",
+    },
+    {
+      name: "learning_format",
+      title: "Format",
+      labelFunction: (key: string) =>
+        key
+          .split("_")
+          .map((word) => capitalize(word))
+          .join("-"),
+    },
+  ].filter((facetSetting) =>
+    (FACETS_BY_CHANNEL_TYPE[channelType] || []).includes(facetSetting.name),
+  )
+}
+
+const getChannelDetails = (field: FieldChannel) => {
+  const channelType = field.channel_type
+  const dataKey = `${channelType}_detail`
+  const fieldData = field as unknown as Record<string, string[] | string>
+  const data = fieldData[dataKey] as unknown as Record<
+    string,
+    string[] | string
+  >
+  return data[channelType]
+}
+const InfoLabel = styled(Typography)(({ theme }) => ({
+  color: theme.custom.colors.mitRed,
+}))
+const ChannelDetailsCard = styled(Box)(({ theme }) => ({
+  border: "1px solid #ccc",
+  borderRadius: "8px",
+  backgroundColor: "white",
+  minWidth: "300px",
+  width: "50%",
+  fontSize: "0.5em",
+  lineHeight: "0.5em",
+  [theme.breakpoints.down("md")]: {
+    width: "100%",
+  },
+}))
+
+const ChannelDetails: React.FC<ChannelDetailsProps> = (props) => {
+  const { field } = props
+  const channelDetails = getChannelDetails(field)
+  const channelType = field.channel_type
+  const channelTitle = field.title
+  const facetManifest = useMemo(
+    () => getFacetManifest(channelType),
+    [channelType],
+  )
+
+  const body = facetManifest.map((value) => {
+    const detailValue = (
+      channelDetails as unknown as { [key: string]: string }
+    )[value.name]
+    if (detailValue) {
+      const label = value?.labelFunction
+        ? value.labelFunction(detailValue, channelTitle)
+        : detailValue
+
+      return (
+        <Box key={value.title} sx={{ margin: "10px" }}>
+          <InfoLabel
+            lineHeight="1.5"
+            fontSize="inherit"
+            variant="h5"
+            gutterBottom
+          >
```

Comment:
The designs look like `variant="subtitle2"` for this one (and no `lineHeight` or `fontSize` needed)